examples/generate_pdfa_with_doctr_output.py

```diff
@@ -0,0 +1,176 @@
+#
+# Lets take a look how to generate a PDF/A document with the doctr output.
+#
+
```

Comment:
Could you remove this file for now please?

Reply:
@fg-mindee 
yes
i have opened a issue for this #562